docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,14 @@
 Version history
 ===============
 
+5.3.1 (2016 December 16)
+-----------------------
+
+Java bug fixes:
```

Comment:
Now we decoupled the low-level components, does this section still make sense /cc @hflynn ?